### DISCUSSION TITLE
German translation file

### DISCUSSION THREAD

**Discussion by herbieherb** [Digitalroot-Valheim/Atokal-AtosArrowsJVL#15] [Ideas]
I translated the language file to german. Unfortunately, the mod wont read it atm.
Maybe you want to include it: [German.zip](https://github.com/Digitalroot-Valheim/Atokal-AtosArrowsJVL/files/10305419/German.zip)

**Comment by Durigans** [comment]
@herbieherb I'm totally new at this world, and at the moment I don't know how to include zip files... Can you help me?

**Comment by Digitalroot** [comment]
How did I miss this. 
I'll add it.
